prime-router/metadata/fhir_mapping/hl7/segments/OBX/Observation.yml

```diff
@@ -0,0 +1,490 @@
+resourceType: Observation
+
+# Not implementing OBX.5 when OBX.2 is RP as the inventory say it still needs to be resolved
+# Not implementing OBX.5 when OBX.2 is CF, the type does not exist in NIST and the mapping is wrong in the inventory
+# Not implementing OBX.5 when OBX.2 is NA (Numeric Array), the inventory does not specify how that should map to SampledData
+# NIST indicates that OBX.10 is a CWE, but Caristix and the HAPI structures are ID
+# OBX.17 is repeatable but maps to method on Observation which is not, so it's all captured in OBXExtension
+# OBX.18 is repeatable but maps to device on Observation which is not, so it's all captured in OBXExtension
+# OBX.20 is repeatable but maps to bodySite on Observation which is not, so it's all captured in OBXExtension
+# Not implementing OBX.22 not in mapping deprecated in NIST
+# Not implementing OBX.26 not in mapping not in NIST
+# Not implementing OBX.27 not in mapping not in NIST
+# Not implementing OBX.28 not in mapping not in NIST
+# Not implementing OBX.29 not in mapping not in NIST
+# Not implementing OBX.31 not in mapping not in NIST
+# Not implementing OBX.32 not in mapping not in NIST
+# Not implementing OBX.33 as it's not in the NIST spec the logic in the inventory cannot be implemented in the library
+
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+code:
```

Comment:
Do the elements without `expressionType: HL7Spec` need conditions to ensure an empty element won't be created when the incoming HL7 field is blank?

Reply:
I believe that a template will not get evaluated if specs come back empty